au4/src/projectscene/view/trackspanel/trackslistmodel.cpp

```diff
@@ -463,8 +463,6 @@ bool TracksListModel::removeRows(int row, int count, const QModelIndex& parent)
 
 void TracksListModel::onProjectChanged()
 {
-    this->disconnectAll();
```

Comment:
So that's what was disconnecting the `selectionController()->trackSelected().onReceive(this, ...)` call made in the ctor.